PyPortal_Azure_Plant_Monitor/azure_gfx_helper.py

```diff
@@ -66,19 +66,19 @@ def __init__(self, is_celsius=True):
         self.moisture_label.y = 135
         self._text_group.append(self.moisture_label)
 
-        self.moisture_text = Label(self.data_font, max_glyphs=10)
+        self.moisture_text = Label(self.data_font, max_glyphs=10, text="")
```

Comment:
This is a workaround for https://github.com/adafruit/Adafruit_CircuitPython_Display_Text/issues/45. This can be reverted once the fix is released.

Reply:
@jimbobbennett  This issue was fixed here: https://github.com/adafruit/Adafruit_CircuitPython_Display_Text/issues/45, could you please update this PR to reflect that?